color_generator/networks/distilbert.py

```diff
@@ -1,26 +1,27 @@
-"""Define DistilBERT network function."""
-from typing import Tuple
+import torch.nn as nn
+from transformers import DistilBertModel, DistilBertConfig
```

Comment:
This could also be done with AutoModel and AutoConfig (https://huggingface.co/transformers/model_doc/auto.html#automodel) to allow different architectures that have the same input and output